apps/health_campaign_field_worker_app/lib/data/local_store/sql_store/tables/individual.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:drift/drift.dart';
```

Comment:
the spec for individual has been updated. Can add those fields in the same or different PR.
https://editor.swagger.io/?url=https://raw.githubusercontent.com/digit-egov/health-api-specs/main/contracts/registries/individual.yml